skills/cmux-diagnostics/SKILL.md

```
@@ -0,0 +1,84 @@
+```
+
+Use `--include-context` only when workspace names, cwd paths, and current cmux identifiers are relevant to the user-reported issue:
+
+```bash
+skills/cmux-diagnostics/scripts/cmux-diagnostics --include-context
+```
+
+## What to Check
+
+1. CLI and socket health:
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)